python/rpdk/go/codegen.py

```diff
@@ -272,3 +272,10 @@ def write_with_relative_path(path):
         for path in (project.root / "internal").rglob("*"):
             if path.is_file():
                 write_with_relative_path(path)
+
+    @staticmethod
+    def _get_plugin_information():
+        return {"plugin-tool-version": __version__, "plugin-name": "go"}
```

Comment:
How's the go plugin runtime version? or how go plugin manage the version?

Reply:
It's added in the go.mod file, for ex. -> https://github.com/brianterry/unicorn-maker/blob/master/go/go.mod#L7. Added code to parse the file and fetch version